tests/release/pipelines/push_to_external_registry.go

```diff
@@ -146,7 +163,7 @@ var _ = framework.ReleasePipelinesSuiteDescribe("[HACBS-1571]test-release-e2e-pu
 						"repository": releasedImagePushRepo,
 					},
 					{
-						"component":  additionalComponentName,
+						"component":  "simple-python",
```

Comment:
This change may be not expected,  Right?  I saw the "additionalCompName" variable above.

Reply:
the constant name used to be “additionalComponentName”, with this change, there is no additionalComponentName. Line 134 gives new value to additionalCompName, so to ensure the component the same as constant additionalComponentName, hard code it here.

Reply:
In case of confusing, I add constant additionalComponentName back